tests/functional/test_stacks_config_prototype.py

```diff
@@ -0,0 +1,95 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests extracted from test_stacks"""
+
+import os
+
+import allure
+import pytest
+from adcm_client.objects import ADCMClient
+from adcm_pytest_plugin import utils
+
+# pylint: disable=W0611, W0621, W0212
+
+
+@allure.step('Load default stack')
+def load_default_stack(client):
+    client.stack.load.update()
+    return client.stack.host.list()
+
+
+@pytest.fixture()
+def client(sdk_client_fs: ADCMClient):
+    return sdk_client_fs.adcm()._api.objects
```

Comment:
`adcm_fs.api` and it is not `client`

Reply:
Actually it is not in use for a while. Will be updated.